dg/lib/cloud_pak_for_data/cpd_4_0_0/types/custom_resource_metadata.py

```diff
@@ -137,8 +137,9 @@ def get_kind_metadata(self) -> KindMetadata:
         KindMetadata
             kind metadata
         """
+        plural = self._kind.lower() + ('s' if not self._kind.lower().endswith('s') else '')
```

Comment:
Add a newline after the docstring (see [coding guidelines](https://github.com/IBM/data-gate-cli/blob/master/docs/coding_guidelines.md#comments))